api/models/Members.js

```diff
@@ -19,6 +19,26 @@
 var bcrypt = require('bcryptjs')
 var is = require('torf')
 
+var hash_password = key => (member, cb) => {
+  if (is.ok(member[key])) {
```

Comment:
This should totally be flipped.

`if (!is.ok(member[key])) return cb(null, member)`

Then the rest of it not nested